app/react-native/src/preview/components/Shared/theme.ts

```diff
@@ -0,0 +1,13 @@
+export const theme = {
```

Comment:
@benoitdion @domyen Can we make the theme vars line up with the web vars? If not use the same utilities?

https://storybook.js.org/docs/configurations/theming/#create-a-theme-quickstart

Reply:
proper theming support is going to be more involved. I'm going to hold off on renaming until we update the ui if that's okay.